### ISSUE TITLE
Use of merqury for illumina paired end reads fot trio binning

### ISSUE THREAD

**Issue by ap1438** [marbl/merqury#91]
Hi, 
i wanted to use tools verkko for de novo assembly using trio binning approach.

I have paired end reads from illumina .
I was wondering how to use the below commands mentioned in the manual of verkko.

meryl count compress k=30 threads=XX memory=YY maternal.*fastq.gz output maternal_compress.k30.meryl
meryl count compress k=30 threads=XX memory=YY paternal.*fastq.gz output paternal_compress.k30.meryl

So should i just cat the R1 and R2 of paired end reads to genetare a single file for both the parents and then run the command.
like cat R1 R2 > Maternal.fastq.gz
cat R1 R2 > Paternal.fastq.gz 
and the use the above command.

Or should i use count forward and count reverse and the union command to generate the maternal_compress.k30.meryl and 
Paternal_compress.k30.meryl.

**Comment by arangrhie** [comment]
Hello @ap1438 ,

The script should work as stated using the latest tip version, or come back in a few days as we are fixing the v1.4 release.
```
meryl count compress k=30 threads=XX memory=YY maternal.*fastq.gz output maternal_compress.k30.meryl
meryl count compress k=30 threads=XX memory=YY paternal.*fastq.gz output paternal_compress.k30.meryl
```
No need to concatenate.
Note memory is in GB. Something like 48 should work fine.

Looping in @brianwalenz here.

**Comment by arangrhie** [comment]
Use the released Meryl v1.4. No need to concatenate R1 and R2, just provide both in line.
